ESLint-Formatter.py

```diff
@@ -38,6 +38,9 @@ def run(self, edit):
 
     output = self.run_script_on_file(self.view.file_name())
 
+    # log output for easier debugging
```

Comment:
I'm not sure that we want to commit this. It seems like it would print lots of messages to the sublime console.